__tests__/feature/editing/selectFromLookup.test.js

```diff
@@ -22,9 +22,9 @@ describe('selecting a value from the lookup modal', () => {
 
     fireEvent.click(addButton)
 
-    await waitFor(() => expect(screen.getByText('Lookup')).toHaveClass('modal-title'))
+    await waitFor(() => expect(screen.getByLabelText('Instance of (lookup)')))
 
-    const input = screen.getByPlaceholderText('Instance of (lookup)')
+    const input = screen.getByLabelText('Instance of (lookup)')
```

Comment:
Additional testing should be added to selectFromLookup to reflect the new functionality.

Reply:
There is no new functionality here. Both before and after the user is able to select a resource from a search. This only changes how the function appears.

Reply:
Even if not testing the tabbing functionality, should be testing that tabs appear. (Does the test currently do this?)

Reply:
I don't have a good idea of how one writes this sort of test (that depends on an external service).  But the existing test should already cover selecting a resource, and with the testing already done by bootstrap-react, we should have sufficient coverage, right?